<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizExamen</class>
 <widget class="QDialog" name="QuizExamen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QRadioButton" name="Answer1">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QRadioButton" name="Answer2">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QRadioButton" name="Answer3">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QRadioButton" name="Answer4">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="QuizTitle">
      <property name="text">
       <string>WElcome To Your Quiz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>635</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="Question">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>What does the abbreviation &quot;OS&quot; stand for?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreviousButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>600</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
